pom.xml

```diff
@@ -61,7 +61,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <!-- <deploy.projectId>[PROJECT ID HERE]</deploy.projectId> -->
```

Comment:
nit: Please revert this change.

Reply:
Done!

Reply:
It's showing as done for me? Weird

Reply:
This is what I see:
https://screenshot.googleplex.com/F02oDCv5tA3.png
But it's not important enough to block submission, tho.

Reply:
Ohhh I misunderstood the request, fixed!

Reply:
Ohhh my bad I misunderstood the request, fixed in latest commit